odl/tomo/geometry/conebeam.py

```diff
@@ -48,75 +49,215 @@ class HelicalConeFlatGeometry(DivergentBeamGeometry, AxisOrientedGeometry):
     and detector positions simultaneously.
 
     In the standard configuration, the rotation axis is ``(0, 0, 1)``,
-    the initial source-to-detector vector is ``(1, 0, 0)``, and the
+    the initial source-to-detector vector is ``(-1, 0, 0)``, and the
```

Comment:
So my requirements were two-fold:

1. Make 2d geometries behave exactly like a z=0 slice of a 3d geometry.
2. Make the initial position as intuitive as possible, in the sense that the detector axes should correspond to axes in the global system, in a reaonable order and not flipped.

Point 1 is done, but for point 2 I agree it currently leads to the slightly less appealing `-x` location of the detector.
What's nice about this variant, though, is that the `(u, v)` detector axes are the `(y, z)` axes of the 3d system since the source sits at `+x` and "looks" at the other axes from the right side.

However, thinking about this, I realize that using `+y` as the initial detector position (i.e. source at -y) would yield `(x, z)` as initial detector axes. That seems to be the better alternative, or what do you think about this suggestion?